README.md

```diff
@@ -29,13 +29,25 @@ Initialize a component with the oc-template-react and follow the CLI instruction
 $ oc init <your-component-name> oc-template-react
 ```
 
-## Extra/Conventions:
+## Extra info:
+### package.json requirements
 - `template.src` - the react App entry point -  should export a react component as `default`
+- `template.type` -  `oc-template-react`
+- required in `devDependencies` -  `oc-template-react-compiler`
+
+### conventions
 - `props = server()` - the viewModel generated by the server is automatically passed to the react application as props
-- `Server Side Rendering` = server side rendering should work as for any other OpenComponent
-- `css-modules` are supported.
 - The oc-client-browser is extended and will now expose all the available react-component at `oc.reactComponents[bundleHash]`
 - React, ReactDOM and PropTypes are not bundled when packaging and publishing the component
```

Comment:
Should we put a note here to explain what does it mean? Just to mention that the oc.require will include them if not in the page already?